src/sparql/qc/general/qc-duplicate-exact-synonym-no-abbrev.sparql

```diff
@@ -31,7 +31,7 @@ SELECT DISTINCT ?entity ?property ?value WHERE {
          oboInOwl:hasSynonymType <http://purl.obolibrary.org/obo/mondo#ABBREVIATION> .
   }
   
-  FILTER NOT EXISTS { ?entity owl:deprecated true }
+  FILTER NOT EXISTS { ?entity1 owl:deprecated true }
```

Comment:
## QC bugfix
I don't 100% understand how the SPARQL query does what it does. Mainly the `VALUES ?property1` and `VALUES ?property2` bit. I guess it's some how able to get all distinct pairs of such cases. But I realize I don't need to fully understand it for the scope of this PR. I see that:

> it checks that no two distinct terms in Mondo can have the same exact synonym.

And I see the bugfix is renaming this variable.